### PR TITLE
Implements Geomap Chart (Fixes #509)

### LINKED ISSUE #509: Create a Map Chart in Clay Charts
In Clay Charts, we'd like to have a comoponent that renders a map by consuming [GeoJSON](http://geojson.org/) data.

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

File: packages/clay-charts/demos/geomap.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+</head>
+<body>
+	<h3>Metal Geomap</h3>
+
+	<div class="flex">
+		<div class="map" id="map"></div>
+	</div>
+
+	<script type="text/javascript">
+		var geomap = new metal.GeomapJSX({
```
Comment (jbalsas) [inline packages/clay-charts/demos/geomap.html]: This doesn't seem to be the jsx demo and still is using the jsx version?

File: packages/clay-charts/src/jsx/Geomap.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import {Geomap} from '../Geomap';
+import {JSXDataManager, JSXRenderer} from 'metal-jsx';
+
+/**
+ * Geomap  component for use with JSX.
+ * @extends Geomap
+ */
+class GeomapJSX extends Geomap {
```
Comment (jbalsas) [inline packages/clay-charts/src/jsx/Geomap.js]: We don't have other classes suffixed `JSX`. I assume this one shouldn't either.

**Comment by jbalsas** [comment]
Hey @Robert-Frampton, mind taking a look at this?

**Comment by jbalsas** [comment]
Hey @julien now that we've merged charts docs.. mind adding one for this so we don't foget? 😉

**Comment by julien** [comment]
@jbalsas yes you're right, I missed that one.

**Comment by Robert-Frampton** [comment]
LGTM

**Comment by julien** [comment]
Pull requested submitted to https://github.com/liferay/clay/pull/525

**Pull request closed without merging**